### PR TITLE
removed braintree/manners

### PR DESCRIPTION
**Pull request by vijayee**
for addressing ipfs/webui issue #28

### CODE REVIEW

**Comment by whyrusleeping** [comment]
:+1: LGTM, lets see if the tests agree

**Comment by jbenet** [comment]
restarted builds

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,4 @@
 
 /test/bin
 .ipfs
+.idea/
```
Comment (jbenet) [inline .gitignore]: i dont think this is supposed to be committed

**Comment by jbenet** [comment]
@vijayee tests pass. LGTM. only one thing:
- [ ] remove the `.gitignore` addition

(ideally amend the same commit and force-push)

**Comment by vijayee** [comment]
I amended that last commit and removed the intellij ignore

**Comment by whyrusleeping** [comment]
@vijayee looks like you added a bunch of intellij stuff?

**Comment by vijayee** [comment]
Yeah I just caught and fixed the intellij garbage.

**Comment by whyrusleeping** [comment]
awesome, @jbenet RFM?

**Comment by jbenet** [comment]
yep, thanks

**Pull request merged**